pyproject.toml

```diff
@@ -62,6 +62,7 @@ dev = [
     "ruff>=0.3.3",
     # Test
     "pytest",
+    "pytest-cov",
```

Comment:
pytest cov is used only in GitHub actions no? we don't really need it for the local dependencies, do we?

Reply:
I actually run it locally often for my other projects. It helps me to see quickly for which parts I need to run tests.

Reply:
ok, let's add it in for now 👍

Reply:
cool